cmd/tusd/cli/serve.go

```diff
@@ -51,8 +59,20 @@ func Serve() {
 
 	SetupPostHooks(handler)
 
+	mux := http.NewServeMux()
+	s := &http.Server{
+		Addr:		address,
+		Handler:        mux,
+		// TODO(rbastic): what to do about these?
```

Comment:
We have a custom timeout implementation because Go's timeout operate on the entire HTTP request and not a single read operation. You can find it here: https://github.com/tus/tusd/blob/master/cmd/tusd/cli/listener.go So I am not sure if we need the timeouts here TBH.